src/App.vue

```diff
@@ -736,9 +736,12 @@ export default class App extends Mixins(
     // NB: will throw if API error
     const userInfo = await AuthServices.fetchUserInfo()
 
-    if (this.isDissolutionFiling) {
+    if (!this.isIncorporationFiling) {
```

Comment:
What if we left out this check? Could an IA proceed without the contact info? Maybe a business question. Please follow up.

Reply:
To create business contact info we need a business and our system will have business once IA completed. We update business contact info from entity-filer once IA is completed. Does that make sense.

Reply:
If I understand correctly, this code is only for dissolution filings.

In this case, I this it will be more clear (and safer) for this condition to be "if dissolution".